salt/metalk8s/orchestrate/backup/files/job.yaml.j2

```diff
@@ -22,15 +22,15 @@ spec:
       containers:
       - name: backup-replication
         image: {{ image }}
-        command:
-          - wget
-          - --accept="*.tar.gz"
-          - --no-host-directories
-          - --mirror
-          - --ca-certificate=/certificate/ca.crt
-          - --user=$(BACKUP_USERNAME)
-          - --password=$(BACKUP_PASSWORD)
-          - https://backup/
+        command: ["/bin/sh", "-c"]
+        args:
+          - >-
+            wget --accept="*.tar.gz"
+            --no-host-directories --mirror
+            --ca-certificate=/certificate/ca.crt
+            --user=$(BACKUP_USERNAME)
+            --password=$(BACKUP_PASSWORD)
+            https://backup/; ls -t -1 /backups/*.tar.gz | tail -n+6 | xargs -t rm -f
```

Comment:
What happens if the wget fail ? It will silently continue and report success on the job no ?

Reply:
both commands can silently fail here,
that is probably why we didn't catch it before
can I add set -e and pipefail ?

Reply:
Hmm yes we could likely, but otherwise just a `&&` instead of `;`